package.json

```diff
@@ -13,18 +13,19 @@
     "@oclif/plugin-help": "^3",
     "@types/jest": "^26.0.3",
     "@types/node": "^14.0.14",
-    "eslint-config-oclif-typescript": "^0.1",
+    "eslint": "^6.8.0",
     "eslint-config-oclif": "^3.1",
+    "eslint-config-oclif-typescript": "^0.1",
     "eslint-config-vtex": "^12.3.0",
-    "eslint": "^6.8.0",
     "globby": "^10",
     "jest": "^25.1.0",
-    "nodemon-notifier-cli": "https://github.com/Slessi/nodemon-notifier-cli.git",
     "nodemon": "^2.0.2",
+    "nodemon-notifier-cli": "https://github.com/Slessi/nodemon-notifier-cli.git",
     "prettier": "^2.0.1",
     "ts-jest": "^25.2.1",
     "ts-node": "^8",
-    "typescript": "^3.8.2"
+    "typescript": "^3.8.2",
+    "vtex": "^2.103.1-beta"
```

Comment:
When we release 'vtex' with the new changes we have to remember to update here

Reply:
Sure